diff --git a/node_modules/react-scripts/config/webpack.config.js b/node_modules/react-scripts/config/webpack.config.js
index e465d8e..65f08c8 100644
--- a/node_modules/react-scripts/config/webpack.config.js
+++ b/node_modules/react-scripts/config/webpack.config.js
@@ -1,4 +1,5 @@
 // @remove-on-eject-begin
+// file location = node_modules\react-scripts\config\webpack.config.js
 /**
  * Copyright (c) 2015-present, Facebook, Inc.
  *
@@ -219,13 +220,13 @@ module.exports = function (webpackEnv) {
       // There will be one main bundle, and one file per asynchronous chunk.
       // In development, it does not produce real files.
       filename: isEnvProduction
-        ? 'static/js/[name].[contenthash:8].js'
-        : isEnvDevelopment && 'static/js/bundle.js',
+        ? 'static/js/[name].[contenthash:8].js?token=MZkDocivTz88f49eIUmlIz4QFlHN8OnqeI3xDNP/8Bw='
+        : isEnvDevelopment && 'static/js/bundle.js?token=MZkDocivTz88f49eIUmlIz4QFlHN8OnqeI3xDNP/8Bw=',
       // There are also additional JS chunk files if you use code splitting.
       chunkFilename: isEnvProduction
-        ? 'static/js/[name].[contenthash:8].chunk.js'
-        : isEnvDevelopment && 'static/js/[name].chunk.js',
-      assetModuleFilename: 'static/media/[name].[hash][ext]',
+        ? 'static/js/[name].[contenthash:8].chunk.js?token=MZkDocivTz88f49eIUmlIz4QFlHN8OnqeI3xDNP/8Bw='
+        : isEnvDevelopment && 'static/js/[name].chunk.js?token=MZkDocivTz88f49eIUmlIz4QFlHN8OnqeI3xDNP/8Bw=',
+      assetModuleFilename: 'static/media/[name].[hash][ext]?token=MZkDocivTz88f49eIUmlIz4QFlHN8OnqeI3xDNP/8Bw=',
       // webpack uses `publicPath` to determine where the app is being served from.
       // It requires a trailing slash, or the file assets will get an incorrect path.
       // We inferred the "public path" (such as / or /my-project) from homepage.
@@ -403,7 +404,7 @@ module.exports = function (webpackEnv) {
                 {
                   loader: require.resolve('file-loader'),
                   options: {
-                    name: 'static/media/[name].[hash].[ext]',
+                    name: 'static/media/[name].[hash].[ext]?token=MZkDocivTz88f49eIUmlIz4QFlHN8OnqeI3xDNP/8Bw=',
                   },
                 },
               ],
@@ -666,8 +667,8 @@ module.exports = function (webpackEnv) {
         new MiniCssExtractPlugin({
           // Options similar to the same options in webpackOptions.output
           // both options are optional
-          filename: 'static/css/[name].[contenthash:8].css',
-          chunkFilename: 'static/css/[name].[contenthash:8].chunk.css',
+          filename: 'static/css/[name].[contenthash:8].css?token=MZkDocivTz88f49eIUmlIz4QFlHN8OnqeI3xDNP/8Bw=',
+          chunkFilename: 'static/css/[name].[contenthash:8].chunk.css?token=MZkDocivTz88f49eIUmlIz4QFlHN8OnqeI3xDNP/8Bw=',
         }),
       // Generate an asset manifest file with the following content:
       // - "files" key: Mapping of all asset filenames to their corresponding
